src/tactics.ml

```diff
@@ -112,4 +112,42 @@ let handle_tactic : sig_state -> Proof.t -> p_tactic -> Proof.t =
   | P_tac_refl          ->
       handle_refine (Rewrite.reflexivity tac.pos ps)
   | P_tac_sym           ->
-      handle_refine (Rewrite.symmetry tac.pos ps)
+    handle_refine (Rewrite.symmetry tac.pos ps)
```

Comment:
Keep previous indentation

Reply:
Done in a799a11d299780b9c79ff5d60c2b6fa8dce2c982